src/msgs/data_requests/types.rs

```diff
@@ -51,21 +51,21 @@ pub struct DataRequest {
     /// Semantic Version String
     pub version:            Version,
     /// Identifier of DR WASM binary
-    pub dr_binary_id:       String,
+    pub exec_program_id:    String,
     /// Inputs for DR WASM binary
     pub dr_inputs:          Bytes,
     /// Identifier of Tally WASM binary
-    pub tally_binary_id:    String,
+    pub tally_program_id:   String,
     /// Inputs for Tally WASM binary
     pub tally_inputs:       Bytes,
     /// Amount of required DR executors
     pub replication_factor: u16,
     /// Filter applied before tally execution
     pub consensus_filter:   Bytes,
     /// Amount of SEDA tokens per gas unit
-    pub gas_price:          U128,
+    pub gas_price:          u64,
```

Comment:
This can/should stay a u128 right? Since it deals with `aseda`

Reply:
These kind of changes need to be updated in the evm contracts too, right?

Reply:
I think it's literally everything that needs to change 😂

Reply:
oops will revert this one my bad.

But yea EVM side needs to be updated as well I wrote a comment on the overlay side for that as well.